views/admin/tpl/email/module_support_email.tpl

```diff
@@ -0,0 +1,181 @@
+          </th>
+          <th class="text-pad">
+            [{oxmultilang ident="GENERAL_ACTIVE"}]
+          </th>
+        </tr>
+        [{foreach from=$emailData.modules item=module}]
+          <tr>
+            <td class="text-pad" style="white-space:nowrap;">
+              [{$module->getId()}]
+            </td>
+            <td class="text-pad">
+              [{$module->getTitle()}]
+            </td>
+            <td class="text-pad">
+              [{$module->getInfo('version')}]
+            </td>
+            <td class="text-pad" style="white-space:nowrap;">
+              [{if $module->isActive()}]
+                [{oxmultilang ident="yes"}]
+              [{/if}]
+            </td>
+          </tr>
+        [{/foreach}]
+      </table>
+    </td>
+  </tr>
+  <tr>
+    <td class="text-pad" colspan="2">&nbsp;</td>
+  </tr>
+  <tr>
+    <td class="text-pad" colspan="2">
+      <b style="white-space:nowrap;">[{oxmultilang ident="payment_method_settings"}]</b>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="2">
+      <table>
+        [{* List all payments *}]
+        [{foreach from=$emailData.payments item=payment}]
+          [{assign var="paymentMethod" value=$payment->getPaymentMethod()}]
+          <tr>
+            <th class="text-pad" style="white-space:nowrap;" colspan="2">
+              [{$payment->oxpayments__oxdesc->value}]
+            </th>
+          </tr>
+          <tr>
+            <td class="text-pad" style="white-space:nowrap;">
+              [{oxmultilang ident="GENERAL_ACTIVE"}]
+            </td>
+            <td class="text-pad">
+              [{if $payment->oxpayments__oxactive->value}]
+                [{oxmultilang ident="yes"}]
+              [{else}]
+                [{oxmultilang ident="no"}]
+              [{/if}]
+            </td>
+          </tr>
+          [{* List all support fields of payment *}]
+          [{foreach from=$paymentMethod->getSupportConfigFields() item=configField}]
+            [{assign var="fieldName" value=$configField.field}]
+            <tr>
+              <td class="text-pad" style="white-space:nowrap;">
+                [{$configField.title}]
+              </td>
+              <td class="text-pad">
+                [{if $configField.type == 'text' }]
+                  [{$payment->$fieldName->value}]
+                [{/if}]
+                [{if $configField.type == 'select'}]
+                  [{foreach from=$configField.options key=optionKey item=optionValue}]
+                    [{if $payment->$fieldName->value == $optionKey}]
+                      [{$optionValue}]
+                    [{/if}]
+                  [{/foreach}]
+                [{/if}]
+              </td>
+            </tr>
+          [{/foreach}]
+          <tr>
+            <td class="text-pad" colspan="2">&nbsp;</td>
+          </tr>
+        [{/foreach}]
+      </table>
+    </td>
+  </tr>
+</table>
+
+</td></tr></table></center></td></tr></table></body></html>
```

Comment:
I think it would be better to include the footer template here.

Reply:
The default footer of the OXID eShop includes some weird social sharing buttons and other unnecessary things for plugin support team.

Reply:
Then we should probably not use the header as well. As far as I understood it, the header is theme-specific, so if a theme modifies it (e.g. by removing the `<center>` tag), our HTML would break.

Reply:
The problem is, header contains CSS definitions needed for the body part, while including theme footer doesn't work well while sent from within admin.